src/components/foodDescription/components/sidebar/Sidebar.js

```diff
@@ -1,27 +1,74 @@
 // Libraries
+import React from 'react';
 import PropTypes from 'prop-types'
 
 // lodash
 import _map from 'lodash/map'
 
+// components
+import Link from '../../../../commonComponents/link/Link';
+
+// constatnts
+import { INITIAL_STATE } from './constants/sidebar.general';
+
 // css
 import './sidebar.css'
 
-function getAnchorJsxFromCategory(category, index){
-    return <a key={category+index} href={`#${category}`}> {category} </a> 
-}
 
-function Sidebar(props){
+class Sidebar extends React.Component{
+
+    constructor(props){
+        super(props);
+        this.state = INITIAL_STATE;
+        this.setCurrentActiveCategory = this.setCurrentActiveCategory.bind(this);
+    }
+
+    componentDidMount(){
+        this.setDefaultActiveState();
+    }
+
+    setDefaultActiveState(){
+        const { categories } = this.props;
+        if(categories.length>0){
+            this.setState({ currentActiveCategory : categories[0] })
```

Comment:
It always sets to the first item in your category, but what if the initial URL is: **http://localhost:3000/#soups** (soups is not the first category)